README.md

```diff
@@ -12,12 +12,14 @@ This service will help service provider to set up DFT/SDE with EDC and EDC as se
 ### Software Version
 
 ```shell
-Helm release version: 1.1.4
-Image version: 1.1.4
+Application version: 1.1.4
```

Comment:
are you sure you don't change the app vrsion after this pr? the file websecuritykeyclaok... is gone; you have code changes

Reply:
Changed the version